guides/source/6_0_release_notes.md

```diff
@@ -178,10 +178,82 @@ Please refer to the [Changelog][action-pack] for detailed changes.
 
 ### Removals
 
+*   Remove deprecated `fragment_cache_key` helper in favor of `combined_fragment_cache_key`.
+    ([Commit](https://github.com/rails/rails/commit/e70d3df7c9b05c129b0fdcca57f66eca316c5cfc))
+
+*   Remove deprecated methods in `ActionDispatch::TestResponse`:
+     - `#success?` in favor of `#successful?`
+     - `#missing?` in favor of `#not_found?`
+     - `#error?` in favor of `#server_error?`
+    ([Commit](https://github.com/rails/rails/commit/13ddc92e079e59a0b894e31bf5bb4fdecbd235d1))
+
 ### Deprecations
 
+*   Deprecate `ActionDispatch::Http::ParameterFilter` in favor of `ActiveSupport::ParameterFilter`.
+    ([Pull Request](https://github.com/rails/rails/pull/34039))
+
+*   Deprecate controller level `force_ssl` in favor of `config.force_ssl`.
+    ([Pull Request](https://github.com/rails/rails/pull/32277))
+
 ### Notable changes
 
+*   Introduce Actionable Errors, allowing the ability to dispatch actions from
```

Comment:
This is internal of the framework so I think we should not include in the changelog.